### DISCUSSION TITLE
Plantilla requisitos del sistema

### DISCUSSION THREAD

**Discussion by AngelCruzO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#32] [Ideas]
Buen día a todos, por lo que he notado a todos nos tomo por sorpresa la plantilla para levantar los requisitos del sistema. 

Aquí les comparto una plantilla sencilla, que con ayuda de información acerca del tema de "Análisis de requerimientos" en el área de software puede ser mas robusta. El contenido de la plantilla solo se basa en requisitos Funcionales y No Funcionales, los cuales son los mínimos para cumplir las necesidades de un cliente.

[Descargar Plantilla](https://docs.google.com/document/d/1fUrUY1DK0YMFX3CoUXyQrI7Bn9lC37f0/edit?usp=sharing&ouid=103632406745328465975&rtpof=true&sd=true)

Espero les sirva de algo y mucha suerte a todos

**Comment by BrawnyGG** [comment]
Muchas gracias! Esta plantilla es mucho más fácil de entender:)

**Reply by AngelCruzO** [reply]
De nada, ya solo es cuestión de investigar y ponerle más contenido o solo asi

**Comment by bryan-vg** [comment]
ea que bien, esta plantilla esta muy entendible, muchas gracias
